dev_tool/src/jumpbox.ts

```diff
@@ -191,6 +194,115 @@ async function ensureAllowlistIP(
     return undefined
 }
 
+// Function to securely prompt for password
+async function promptPassword(prompt: string): Promise<string> {
+    const rl = createInterface({ input, output })
+
+    return new Promise((resolve) => {
+        const stdin = process.stdin
+        const listener = (char: Buffer) => {
+            const charStr = char.toString()
+            switch (charStr) {
+                case '\n':
+                case '\r':
+                case '\u0004':
+                    stdin.removeListener('data', listener)
+                    stdin.setRawMode(false)
+                    stdin.pause()
+                    output.write('\n')
+                    rl.close()
+                    resolve(password)
+                    break
+                case '\u0003': // Ctrl+C
+                    stdin.removeListener('data', listener)
+                    stdin.setRawMode(false)
+                    stdin.pause()
+                    rl.close()
+                    process.exit()
+                    break
+                case '\u007f': // Backspace
+                    if (password.length > 0) {
+                        password = password.slice(0, -1)
+                        output.write('\b \b')
+                    }
+                    break
+                default:
+                    password += charStr
+                    break
+            }
+        }
+
+        let password = ''
+        output.write(prompt)
```

Comment:
One more thing, could we hide the input for the password?

Reply:
From our 1:1: @mojotalantikite and I thought it would be worthwhile to implement masking the password input in this PR. We also talked about how team members work in public spaces and trying to use the jumpbox in those spaces is bad practice so @mojotalantikite Could you add that caution to the docs?